pkg/ebpf/c/tracee.bpf.c

```diff
@@ -950,8 +946,7 @@ statfunc int check_is_proc_modules_hooked(program_data_t *p)
         pos = n;
         n = list_next_entry_ebpf(n, list);
         if (&pos->list == head) {
-            ret = 0;
-            break;
+            return 0;
```

Comment:
Should be `MOD_NOT_HIDDEN`?

Reply:
No.. not hidden refers to a specific module. This just indicates the search is over